### ISSUE TITLE
The 'Plugin#Exit' event is documented but appears to lack implementation

### ISSUE THREAD

**Issue by mesr** [postcss/postcss#1840]
Despite the `Plugin#Exit` listener being [documented](https://postcss.org/api/#plugin-exit) in the plugin API page. implementing it makes the PostCSS engine crash with the following error message:

```
Error: Unknown event Exit in postcss-myplugin. Try to update PostCSS (8.4.23 now).
    at LazyResult.prepareVisitors (/usr/lib/node_modules/postcss-cli/node_modules/postcss/lib/lazy-result.js:456:19)
    at LazyResult.runAsync (/usr/lib/node_modules/postcss-cli/node_modules/postcss/lib/lazy-result.js:403:10)
    at async Promise.all (index 0)
```

The issue is easily replicated using the following plugin code:
```javascript
module.exports = (opts = {}) => {
    return {
        postcssPlugin: 'postcss-myplugin',
        Exit() {}
    }
}
module.exports.postcss = true
```

**Comment by ai** [comment]
It is a docs issue. We have a `RootExit` (at the end of processing the root, but will call again on every root changes) and `OnceExit` (the same but called only once).

Fixed 87fcd63e

Do you have any reasons why you may want `Exit` over `RootExit` or `OnceExit`?

**Comment by mesr** [comment]
Thank you for your quick answer! I came across this issue while trying to find a way to implement final touch-ups to the AST without triggering any further processing by other plugins, regardless of what plugins are configured. I was hoping that the `Plugin#Exit` might be the answer.

**Comment by ai** [comment]
Yes, `Plugin#OnceExit` is the way. I will release types fix tomorrow.

**Comment by ai** [comment]
The fix was released in 8.4.24.
